terms/structuralriskminimization_maf.tex

```diff
@@ -0,0 +1,99 @@
+
+
+\textbf{Theorem}: For any $\delta$ and distribution $D$, with probability at least $1-\delta$ the output of the SRM procedure satisfies 
+\begin{equation}\label{thm}
+\centering
+\[R(\hat{f_{i}})\leq\underset{k\in\mathbb{N}}{min}\left\{ \underset{f\in S_{k}}{min}R(f)+2\epsilon_{k}\left(n,w(k)\delta\right)\right\} \]
+\end{equation}
+
+Due to sampling issues, if the algorithm were to select the set $S_{i}$ with the lowest true error bound, it could technically return different sets each time. For this reason we use the above theorem that tells us that we can guarantee nearly the same bound as would apply on the smallest set containing our output $f$. 
+
+
+Below is a diagram illustrating SRM:
+
+\begin{figure*}
+\centering
+	\includegraphics[scale=0.5]{SRMDiagram.png}
+	\caption{An illustration of the empirical risk and confidence for different functions. The optimal function is found at the minimum of the bound, listed as $f^{*}$}
+  	\label{fig:srm}
+\end{figure*}
+
+\section{Examples}
+
+
+Consider a simple machine learning problem, where we need to find the ideal function to represent our model. As mentioned above, to carry out SRM we have two parts to our procedure, the first of which is finding the function $f_{i}$ that returns to us the minimum risk $R(f)$ for each structure $S_{i}$. The definition of our risk function depends on the type of problem we are solving. For a classification problem, one may want to use a zero-one loss function or a hinge-loss function. On the other hand, for a regression problem, a Huber loss function may be favourable. Once we have selected a suitable loss function, our next step is to look at functions to represent our capacity measure. The $l_{1}$-regularizer and $l_{2}$-regularizer are two examples of functions that can be used to calculate the capacity measure. 
+
+\
+
+Below is the formula that needs to be solved for a support vector machine, according to SRM \cite{srmneumann}:
+
+\begin{equation}\label{srm2}
+\centering
+\[\underset{f}{min}\left[C\cdot\sum_{i=1}^{n}{max}\left[\left|y_{i}-f(x_{i})\right|,0\right]+\left\Vert \mathbf{w}\right\Vert _{2}^{2}\right]\]
+\end{equation}
+
+with the solution $\hat{f}(x_{i})=C_{k}\cdot K(x,x_{k})$ where $K$ represents the kernel function. Notice how SVMs use the hinge-loss function as their risk function and the $l_{2}$-regularizer to quantify complexity. 
+
+If we are to carry this out for linear regression, our formula would
+look something like this:
+
+\begin{equation}\label{srm3}
+\centering
+\[\underset{f}{min}\left[\sum_{i=1}^{n}\left(f(x_{i})-y_{i}\right)^{2}+\left\Vert \mathbf{w}\right\Vert _{2}^{2}\right]\]
+\end{equation}
+
+In reality, we can use whichever regularizer we prefer, however, the $l_{2}$-regularizer has an advantage over the $l_{1}$-regularizer as it is strictly convex and differentiable everywhere.
+\index{class options|)}
```

Comment:
"class options" in index ??? why?  this is to include multiple pages in an index and should match the starting tag.